demo/backend/navigation/navigator/bottom-tab/tab-1.xml.njk

```diff
@@ -25,10 +29,10 @@ hv_button_behavior: "back"
       target="tab-2"
```

Comment:
switch to href

Reply:
[bdba9be](https://github.com/Instawork/hyperview/pull/969/commits/bdba9be7457df7c236cfc745aa08ef9f318dbeb8)